services/devicestorage/acl.json

```diff
@@ -1,6 +1,33 @@
 {
-  "allowedEverybody": {
-    "allowedOrigins": [".*"],
-    "rules": {}
+  "Internal Tests": {
+    "allowedOrigins": [
+      "^https://.*\\.telefonica\\.com/WebAPI_pf.*",
+      "^https://.*\\.github\\.io/WebAPI_pf.*"
+    ],
+    "rules": {
+      "onchange": [],
+      "enumerateEditable": [],
+      "enumerate": [],
+      "usedSpace": [],
+      "getEditable": [],
+      "get": [],
+      "freeSpace": [],
+      "delete": [],
+      "available": [],
+      "addNamed": [],
+      "add": [],
+      "getDeviceStorage": [{
+        "permissions": {
+          "apps": "readonly",
+          "music": "readonly",
+          "pictures": "readonly",
+          "sdcard": "readonly",
+          "videos": "readonly"
+        }
```

Comment:
The main constraint is the r/w permission. Once the permission is assigned, some kinds of operations such as `getEditable`, `add`, etc. must be blocked if the client doesn't have enough permissions